Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.cpp

```diff
@@ -352,11 +352,13 @@ void WebSWServerToContextConnection::startFetch(ServiceWorkerFetchTask& task)
 
 void WebSWServerToContextConnection::didReceiveFetchTaskMessage(IPC::Connection& connection, IPC::Decoder& decoder)
 {
-    RefPtr fetch = m_ongoingFetches.get(ObjectIdentifier<FetchIdentifierType>(decoder.destinationID())).get();
-    if (!fetch)
-        return;
+    if (ObjectIdentifier<FetchIdentifierType>::isValidIdentifier(decoder.destinationID())) {
```

Comment:
We should probably turn this into a MESSAGE_CHECK.
We don't need to gracefully fail when web content sends us an invalid identifier.